OpenSim/Simulation/Model/Smith2016Ligament.h

```diff
@@ -0,0 +1,277 @@
+pennation angle of 90 degrees is approached (\f$\phi < 90^\circ\f$), and the
+fiber will also stop contracting as its length approaches a lower bound
+(\f$ \hat{l}_{CE} > lowerbound\f$), which is typically around half the fiber's
+resting length (to ensure \f$ \mathbf{f}_L(\hat{l}_{CE}) > 0 \f$). The fiber is
+prevented from reaching unphysiological lengths or its maximum pennation angle
+using a unilateral constraint. Additionally, the force-velocity curve is
+modified so that it is invertible.
+When an elastic tendon without fiber damping is selected, the minimum
+active-force-length value is set to 0.1, the minimum permissible activation is
+set to 0.01, and the maximum permissible pennation angle is set to acos(0.1) or
+84.3 degrees. This is done as a convenience for the user to prevent the model
+from taking an unreasonable amount of time to simulate.
+<B>(Rigid Tendon) or (Elastic Tendon with Fiber Damping)</B>
+Neither of these formulations has any singularities. The lower bound of the
+active-force-length curve can be zero (min( \f$ \mathbf{f}_L(\hat{l}_{CE})) = 0
+\f$), activation can be zero (i.e., the muscle can be turned off completely),
+and the force-velocity curve need not be invertible.
+The rigid tendon formulation removes the singularities by ignoring the
+elasticity of the tendon. This assumption is reasonable for many muscles, but it
+is up to the user to determine whether this assumption is valid.
+The formulation that uses an elastic tendon with fiber damping removes
+singularities by solving the equilibrium equation with Newton's method. This is
+possible because the partial derivative of the equilibrium equation with respect
+to fiber velocity is always positive if \f$ \beta > 0\f$ and, thus, Newton's
+method can find a solution to the equilibrium equation.
+When either of these singularity-free formulations is selected, the minimum
+active-force-length value and the minimum permissible activation are set to
+zero. This is done as a convenience for the user, as these changes make the
+results of the model more realistic yet incur no performance penalty. The
+maximum pennation angle is left as acos(0.1) or 84.3 degrees, as allowing higher
+pennation angles results in an increasingly stiff fiber velocity state as
+pennation angle increases.
+
+<B>Reference</B>
+	Smith, C.R., Lenhart, R.L., Kaiser, J., Vignos, M.F. and Thelen, D.G.,
+	(2016). Influence of ligament properties on tibiofemoral mechanics
+	in walking. J Knee Surg, 29(02), 99-106.
+	
+	Wismans, J.A.C., Veldpaus, F., Janssen, J., Huson, A. and Struben, P., 
+	(1980). A three-dimensional mathematical model of the knee-joint. 
+	J Biomech, 13(8), 677-685.\n\n"
+
+	Blankevoort, L. and Huiskes, R., (1991).
+	Ligament-bone interaction in a three-dimensional model of the knee.
+	J Biomech Eng, 113(3), 263-269
+@author Colin Smith
+
+*/
+
+	class OSIMSIMULATION_API Smith2016Ligament : public Force  {
```

Comment:
Use spaces instead of tabs. This is a setting in Visual Studio. There's also something called Retabify in Visual Studio.